docs/registry/rhoas-cli-getting-started-registry/README.adoc

```diff
@@ -373,6 +375,103 @@ For more information about any of the options, view the command help `rhoas serv
 ====
 --
 
+[id="proc-configuring-service-registry-rules_{context}"]
+== Configuring {registry} rules
+
+[role="_abstract"]
+To prevent invalid content from being added to {registry}, you can configure optional rules to check the artifact content. Global rules apply to all artifacts in a particular {registry} instance. Configured artifact-specific rules override any configured global rules. Before a new artifact version can be uploaded to the registry, all configured global rules or artifact-specific rules must pass.
+
```

Comment:
Suggest "Global rules apply to all artifacts in a particular {registry} instance.  Artifact rules apply to the specified artifact only. Configured artifact rules override any configured global rules..." to match the current UI and docs.

Reply:
Done :)